dynamiqs/utils/states.py

```diff
@@ -22,6 +22,16 @@ def fock(
 ) -> Tensor:
     """Returns the ket of a Fock state or the ket of a tensor product of Fock states.
 
+    Args:
+        dims: Dimension of the Hilbert space of each mode.
+        states: Fock state of each mode.
+        dtype: Data type of the returned tensor.
+        device: Device of the returned tensor.
+
+    Returns:
+        Tensor of size _(n, 1)_ : Ket of the Fock state or tensor product of Fock
+            states.
```

Comment:
I'd suggest reverting to the previous format.

Reply:
Is it fine if this is in a separate PR? I did some doc updates in this one, but it's starting to be messy (sorry for that).